gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -98,6 +98,13 @@ should_skip_page (GisDriver    *driver,
 {
   guint i = 0;
 
+  /* check the Hack special case first */
+  if (gis_driver_is_hack (driver)) {
+    if (g_strcmp0 ("eula", page_id) == 0 || g_strcmp0 ("endless_eula", page_id) == 0) {
```

Comment:
You don't need to worry about the `eula` page, we always disable it.
I'd rather disable the `endless_eula` page in the way we disable the others by doing it directly in `gis_prepare_endless_eula_page()`